packages/eslint-plugin/tests/rules/no-base-to-string.test.ts

```diff
@@ -51,6 +51,11 @@ ruleTester.run('no-base-to-string', rule, {
     // toString()
     ...literalListWrapped.map(i => `${i === '1' ? `(${i})` : i}.toString();`),
 
+    // toLocalString()
```

Comment:
```suggestion
    // toLocaleString()
```

Here and below - unless I missed something?

Reply:
Oops, I fixed it in [31a7cf4](https://github.com/typescript-eslint/typescript-eslint/pull/10138/commits/31a7cf4d164836c8613eb10f2db06dba15f9d16e).